database/migrations/2016_09_06_134433_add_sticky_to_topics_tables.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddStickyToTopicsTables extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table('topics', function (Blueprint $table) {
+			$table->tinyInteger('sticky')->unsigned()->default(0)->index();
```

Comment:
Laravel has a nice `boolean` method which is cleaner for databases that support boolean columns ;)